packages/styles/settings-tools/_u.preview-grid.scss

```diff
@@ -0,0 +1,33 @@
+$grid-line-color: rgba(0, 100, 100, 0.1);
+$grid-subline-color: rgba(0, 100, 100, 0.03);
+$grid-shadow-color: #ececec;
+$grid-magic-unit: #{$grid-magic-unit}rem;
+
+.mu-preview-grid {
```

Comment:
maybe just apply the bg directly to the class, because setting position relative on the user dom may cause him problems

Reply:
fixed